### PR TITLE
chore(observability): revert internal tracing proof of concept

### PR DESCRIPTION
**Pull request by lukesteensen**
This reverts commit 37e06de830377f1ded73d70014f099ae487ad6da (#7929).

Signed-off-by: Luke Steensen <luke.steensen@gmail.com>

### CODE REVIEW

File: src/app.rs [hunk 1]
```diff
@@ -145,6 +132,11 @@ impl Application {
                     return Err(code);
                 };
 
+                info!(message = "Log level is enabled.", level = ?level);
```
Comment (jszwedko) [inline src/app.rs:135]: Did you mean to re-add these lines? They seem unrelated to the commit being reverted.

**Review by jszwedko — APPROVED** [review]
👍 it was good experiment. It seems like the revert wasn't exactly clean (see the below comment).

File: src/app.rs (hunk 1 above)
Comment (lukesteensen) [reply src/app.rs:135]: This was moved from above, where it needed to happen in a `rt.block_on` when we had the span shipper as part of `trace::init`. Definitely looks a bit different though 😄

Comment (jszwedko) [reply src/app.rs:135]: Aha, yes I see where it was before 👍

**Pull request merged**